### PR TITLE
Fixed unit tests for Dokany 1.0.0-RC4

### PR DESCRIPTION
**Pull request by viciousviper**
Squashed commits for adaptation of the unit tests project to dokany 1.0.0-RC4

### CODE REVIEW

File: DokanNet/DokanOptions.cs [hunk 1]
```diff
@@ -10,7 +10,10 @@ namespace DokanNet
     [Flags]
     public enum DokanOptions : long
     {
-        /// <summary>Enable output debug message.</summary>
+        /// <summary>Fixed Drive</summary>
+        FixedDrive = 0,
+
+        /// <summary>Enable ouput debug message</summary>
```
Comment (magol) [inline DokanNet/DokanOptions.cs]: Missing point in the end of the sentence.

Comment (Liryna) [inline DokanNet/DokanOptions.cs]: fixed https://github.com/dokan-dev/dokan-dotnet/blob/master/DokanNet/DokanOptions.cs :smile: (merged too fast)

**Comment by Liryna** [comment]
Thank you @viciousviper I have merged your commit here https://github.com/dokan-dev/dokan-dotnet/commit/ca783d07de6b9000362704c4301b742392537d5f

Very appricate fix ^^ !

**Comment by Liryna** [comment]
build now failing :cry: 
https://ci.appveyor.com/project/Liryna/dokan-dotnet/build/job/9rieubogvo25qelu

**Comment by viciousviper** [comment]
Yeah, the wonders of git :-(

**Comment by viciousviper** [comment]
I'm on it. Looks to be just a minor nuisance.

I'll suggest a solution for the .NET 4.0 vs. .NET 4.6 situation as well. Just give me a couple of minutes.

**Pull request closed without merging**